src/aiida_pseudo/cli/install.py

```diff
@@ -297,91 +438,65 @@ def cmd_install_pseudo_dojo(
         'upf': UpfData,
     }
 
-    # yapf: disable
-    paw_configurations = (
-        PseudoDojoConfiguration('1.0', 'PBE', 'SR', 'standard', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'PBE', 'SR', 'stringent', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'LDA', 'SR', 'standard', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'LDA', 'SR', 'stringent', 'jthxml')
-    )
-    # yapf: enable
-
     try:
         pseudo_type = pseudo_type_mapping[pseudo_format]
```

Comment:
Not introduced by this PR, but this try/except is not necessary since the correct values are already validated by the click option, which is a `Choice`